testsuite/features/secondary/srv_menu.feature

```diff
@@ -181,72 +181,17 @@ Feature: Web UI - Main landing page menu, texts and links
   Scenario: Sidebar link destination for Systems
```

Comment:
WOW.
What a big clean-up did we make in these scenarios!
If those entries exist somewhere, instead of delete them, we need to navigate properly to the new location.

Reply:
Oh apparently those pages will not exist anymore, anywhere? 😮

Reply:
Clarified by Cédric.